core/src/test/java/org/mobilitydata/gtfsvalidator/notice/sample/AnotherTestValidationNotice.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice.sample;
+
+import com.google.common.collect.ImmutableMap;
+import org.mobilitydata.gtfsvalidator.annotation.SchemaExport;
+import org.mobilitydata.gtfsvalidator.notice.SeverityLevel;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.type.GtfsColor;
+import org.mobilitydata.gtfsvalidator.type.GtfsDate;
+import org.mobilitydata.gtfsvalidator.type.GtfsTime;
+
+/**
+ * {@code ValidationNotice} defined for tests purpose.
+ */
+public class AnotherTestValidationNotice extends ValidationNotice {
+
+  @SchemaExport
+  public AnotherTestValidationNotice(String filename, long csvRowNumber, String fieldName,
+      Object fieldValue, double otherFieldValue, GtfsDate sampleDate, GtfsTime sampleTime,
+      GtfsColor sampleColor) {
+    super(new ImmutableMap.Builder<String, Object>()
```

Comment:
Is there a reason why you are calling a `new ImmutableMap.Builder<>()` here instead of a `ImmutableMap.of()` like in `SomeTestValidationNotice.java` and `SampleTestValidator.java`? I would keep it this way if there is a reason behind this choice, but if it's arbitrary I'd change it to keep the consistency between the sample files.

Reply:
As discussed, for some reason the method `ImmutableMap.of()` does not allow to create a map with `GtfsDate`, or `GtfsTime` values, hence the recourse to `new ImmutableMap.Builder<>()`. For the sakes of consistency, I'll change the constructors (for `SomeTestValidationNotice `, `AnotherTestValidationNotice` and `NestedTestValidatorNotice`) to use `new ImmutableMap.Builder<>()`.